frontend/src/pages/AccountSettingPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { Divider, Grid } from "@mui/material";
+
+import { NavigationBar } from "../components/NavigationBar";
+
+export const AccountSettingPage = () => {
+  return (
+    <Grid container>
+      <Grid item xs={1}>
+        <NavigationBar />
```

Comment:
Instead of having pages import the `NavigationBar`, its better to implement a `SidebarLayout` with the `NavigationBar` and an `Outlet` to render the content of the pages.

This makes the code more DRY.